### ISSUE TITLE
KMS ID or KMS ARN

### ISSUE THREAD

**Issue by fliphess** [awslabs/aurora-snapshot-tool#31]
Hi! 

I'm having a hard time getting this tool to work as it keeps giving errors about kms keys not being found, but I am not completely sure how to debug such errors. 

What values are required for setting up the KMS key configuration?  Should I use the KMS Key ID or the KMS Key ARN?

**Comment by egm7** [comment]
TLDR;

- Don't use the default 'aws/rds' key on cluster. Customer Managed Key (CMK) is appropriate.
- Setup share of CMK if across account is needed.
- Use full ARN in KmsKeySource if across account is needed.

I hit a similar issue with the "KMSKeyNotAccessibleFault" error in the lambda logs. After updating the Aurora cluster with a CMK instead of the default aws/rds key I was able to share the CMK with since I had to share across account. 

This guide is shows how to share. 
https://docs.aws.amazon.com/kms/latest/developerguide/key-policy-modifying-external-accounts.html

After doing these steps the error persisted and I discovered that the "KmsKeySource" I had entered was simply the Key ID and since I was sharing across accounts I needed to specify the ARN since it contains the local (origin) account ID. 

Hope this helps.
